frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseSlideInBelowAction.java

```diff
@@ -144,14 +144,10 @@ public void actionPerformed(AnActionEvent anActionEvent) {
         gitRepository,
         branchLayout,
         branchLayoutWriter,
+        getGraphTable(anActionEvent),
         preSlideInRunnable,
         slideInOptions,
-        parentName) {
-      @Override
-      public void onFinished() {
-        getGraphTable(anActionEvent).queueRepositoryUpdateAndModelRefresh();
```

Comment:
we no longer need it as `enableEnqueuingUpdates` will trigger the update, right?

Reply:
I hope so 😅